src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -505,5 +506,58 @@ public async void UpdateReviewBackground()
                 }
             }
         }
+
+        public async Task<string> GetApiDiffFromAutomaticReview(CodeFile codeFile)
```

Comment:
Should this logic be shared with the other APIs that are checking to see if a API is approved or not?